demo/mobile/src/app/services/userService.js

```diff
@@ -18,8 +18,13 @@ UserService.prototype.readUser = function readUser(userId) {
   });
 };
 
+
+UserService.prototype.getProfile = function(userId) {
+  return this.readUser(userId);
+};
+
 UserService.prototype.listUsers = function listUsers() {
-  return Promise.resolve([]);
+  return Promise.all(this.readUser(userId));
```

Comment:
Ah, using `Bluebird.all` to wrap the results in an array? That's actually nice  :+1: